content/en/observability_pipelines/setup_opw.md

```diff
@@ -62,10 +62,22 @@ To set bootstrap options, do one of the following:
 : env var: `DD_OP_THREADS`
 : The number of threads to use for processing (optional, default: the number of available cores).
 
+`proxy`
+: env variables: `DD_HTTP_PROXY`, `DD_HTTPS_PROXY`
```

Comment:
Let's name them exactly as the agent names them [here](https://docs.datadoghq.com/agent/configuration/proxy/?tab=linux#environment-variables).

E.g. rename `DD_HTTP_PROXY` to `DD_PROXY_HTTP`

Reply:
Fixed in this [commit](https://github.com/DataDog/documentation/pull/23902/commits/9ee5cfa303027aeb03072376af68c88de4128557).